pkgs/by-name/ca/cargo-gg/package.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitLab,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-gg";
+  version = "0.1.1";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.scd31.com";
+    owner = "stephen";
+    repo = "cargo-gg";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-SnxMY8HDVIVp6zNGFiGZDJK0B76rktKhOw/Xtad96SA=";
+  };
+
+  cargoHash = "sha256-WVjM7z6blk4CBO3FpzXfiiopu/lsTnzt2XnreLizMKM=";
```

```suggestion
  useFetchCargoVendor = true;
  cargoHash = "<the-new-hash-here>";
```

the current default fetcher is in the process of being phased out due to cargo changing its implementation, thus breaking all FOD hashes.
`useFetchCargoVendor` lets you opt into the new, more stable fetcher.
Don't forget to update the hash!

(the boolean flag will be removed once it becomes the new default)